### DISCUSSION TITLE
Update 2.3.4 – more mess

### DISCUSSION THREAD

**Discussion by ZeoSonE** [rsayfutdinov/Pikka#19] [Q&A]
Hi there,
update 2.3.4 brought even more mess. This is what I have now
<img width="995" alt="Screenshot 2021-07-13 at 20 58 19" src="https://user-images.githubusercontent.com/7634694/125517197-1854315a-e94a-46f4-a508-360786428669.png">
Sections duplication, colours in same titled sections are different, many random duplications of a colours.

Total mess

**Comment by rsayfutdinov** [comment]
Please try to use "Settings->Migrate" button and then restart the app as suggested before. This should bring your library to the same state as before v.2.3.3

**Comment by ZeoSonE** [comment]
I have done this right after the update, two days ago

On 13 Jul 2021, at 21:22, ruslan ***@***.***> wrote:

﻿
Please try to use "Settings->Migrate" button and then restart the app as suggested before. This should bring your library to the same state as before v.2.3.3
